docs/tutorials/google/advanced_calibration.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<p>This is strange, the results are unexpected. Does your branch include https://github.com/quantumlib/Cirq/pull/4092 which contain a fix for poor XEB performance?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>That was in the v0.11 release, so yes it should. (My branch is just cirq==0.11.0.)</p>